src/conversions.jl

```diff
@@ -5,6 +5,29 @@ export rescale
 
 using ..LeapSeconds
 
+"""
+    tttai(jd1, jd2)
```

Comment:
Docstring formatting is not quite right. Have a look at the other docstrings...but I will merge anyways 🎉